xbmc/interfaces/legacy/Addon.h

```diff
@@ -97,12 +97,158 @@ namespace XBMCAddon
       String getLocalizedString(int id);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_xbmcaddon
+      /// @brief \python_func{ xbmcaddon.Addon([id]).supportsInstanceSettings() }
+      /// Returns true if asked add-on supports instances
```

Comment:
```suggestion
      /// Returns true if the requested add-on supports several instances
```

An add-on with a single instance still supports instances (but just one :) )